handlers.go

```diff
@@ -409,7 +409,10 @@ func (t *TricksterHandler) buildRequestContext(w http.ResponseWriter, r *http.Re
 	ctx.Origin.OriginURL += strings.Replace(ctx.Origin.APIPath+"/", "//", "/", 1)
 
 	// Get the params from the User request so we can inspect them and pass on to prometheus
-	ctx.RequestParams = r.URL.Query()
+	if err := r.ParseForm(); err != nil {
+		return nil, err
```

Comment:
All the other errors in this method are `fmt.Errorf()`-wrapped to add context, would be nice to do the same here.

Reply:
@juliusv I'll do that, but It seems like this is a perfect use-case for https://github.com/pkg/errors :)